### DISCUSSION TITLE
How to make a website without edit extension?

### DISCUSSION THREAD

**Discussion by HelloLudger** [datenstrom/community#606] [Ask a question] [answered]
Hi,

is this possible in any standard way?
I would prefer to only develop the site locally and push the changes to the webserver.

I deleted the edit. php/css/js/woff files from /system/extensions and the Edit.... entries in yellow-system.ini.

It seems to work, but here are probably more places which should be updated. Is there a "local only" installation I don't know about?

Thanky, I love the simplicity!

**Comment by schulle4u** [comment, marked as answer]
Yes, Yellow can be used locally as a static site generator from the [command line](https://github.com/datenstrom/yellow-extensions/tree/master/source/command). You don't need to remove the edit extension, AFAIK it won't be included in your static site. However Yellow has a very convenient extensions management from the command line. If needed you could add or remove the edit extension simply by typing

`~$ php yellow.php install edit`

`~$ php yellow.php uninstall edit`

**Reply by HelloLudger** [reply]
Thank you, just to clarify:
The "installation wizard" after unpacking just installs the desired extension (blog/wiki), if  I chose "website" no extension is installed?

**Reply by schulle4u** [reply]
This is correct. The "Website" option only installs some example data, depending on language selection. After successful installation the `install-*.zip` files will be removed from your `system/extensions` folder.
